src/style.scss

```diff
@@ -5,4 +5,19 @@
   --color-black: #000;
   --color-seashel: #f1f1f1;
   --color-silver: #ccc;
+}
+
+html, body {
+  height: 100%;
+
+  * {
```

Comment:
* means select everything
it has no place in the html, body selector, it should be outside of it.

Reply:
aaa, now I understand

Reply:
So, as I did have no idea how I can do it, I looked up on google and I've found exactly on w3schools it does say about addind a div element for height. Here is the link where I've found it.
![image](https://user-images.githubusercontent.com/46096197/51432571-bfe5e280-1c31-11e9-9ab3-028c5887ec61.png)